src/wazuh_modules/vulnerability_scanner/qa/test_data/008/Readme.md

```diff
@@ -0,0 +1,17 @@
+# Test description
+
+- Vulnerability detection validation, vulnerability remotion after upgrade and entry deletion after `integrity_clear` event.
+
+## Platforms
+
+- Microsoft Windows 10 19045.4043
+- Microsoft Windows 10 19045.4046
+
+## Checks performed
+
+- CVE-2024-23224 does affect `MacOS` OS version `14.0`.
```

Comment:
We need to update the `macOS` references in this Windows test.

Reply:
Oops!